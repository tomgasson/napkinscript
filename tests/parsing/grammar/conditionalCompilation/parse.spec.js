runParser(__dirname, false, false, { BACKEND: "native" });
