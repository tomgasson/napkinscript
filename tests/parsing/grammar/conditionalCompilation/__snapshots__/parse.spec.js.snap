// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`directive.js 1`] = `
";;Js.log \\"native backend\\"
;;Js.log \\"native backend\\"
;;Js.log \\"== native\\"
;;Js.log \\"byte backend\\"
;;Js.log \\"byte backend\\"
;;Js.log \\"it is native backend\\""
`;
